<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Question</class>
 <widget class="QDialog" name="Question">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="question_label">
       <property name="font">
        <font>
         <family>AcadEref</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>十六进制数12H的二进制的表示为?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="question_input"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="question_ok">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="question_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
